geopandas/base.py

```diff
@@ -1228,7 +1234,6 @@ def geom_equals_exact(self, other, tolerance, align=True):
         See also
         --------
         GeoSeries.geom_equals
-        GeoSeries.geom_almost_equals
```

Comment:
It would be good to also update the docstring for the tolerance parameter as it is currently incorrect, it is a relative error tolerance, rather decimal place precision.